src/modals/Menu.tsx

```diff
@@ -133,6 +139,32 @@ const ThemeSwitcher = () => {
   );
 };
 
+const NotificationToggle = () => {
+  const { t } = useTranslation();
+  const { theme } = useContext(ThemeContext);
+  const { notificationsStatus, updateNotificationsStatus } =
+    useContext(NotificationsContext);
```

Comment:
This contezt only stores boolean flag, so maybe it's better to use NotificationEnabledContext instead?